### PR TITLE
place to draft forum post - DO NOT MERGE TO MASTER

### PR DESCRIPTION
**Pull request by mrjones-plip**
The first half of #441 is [docs](https://github.com/medic/cht-docs/pull/449) (out for final review!) and the second half is to add a forum post that documents a moment in time on literal steps on how to deploy an offline CHT server instance.  Since there might be some input about how to best do this, I figured I'll do a slightly awkward thing of creating a docs PR of the forum post with no intention of merging it - just publishing it on forum and closing/deleting the PR.

Here's a [handy rendered version](https://github.com/medic/cht-docs/blob/441-DO-NOT-MERGE-forum-post-draft/content/en/apps/guides/forum-offline.md) which is a lot easier to read than the [PR's diff](https://github.com/medic/cht-docs/pull/456/files).

per #441

### CODE REVIEW

**Comment by mrjones-plip** [comment]
@medic/site-reliability-engineering - anybody interested in reviewing this?  While I need to run through it one more time from scratch, I've pretty well tested it so feel comfy asking for input.  @Hareet  - you mentioned  you were interested!  Here it is when you get back from your time off :beach_umbrella: 

CC @abbyad @michaelkohn @garethbowen

**Comment by mrjones-plip** [comment]
Doesn't seem like anyone has looked at this just yet, but I did a pretty major refactor of the content to reduce the number of steps and simplify the approach.  Just went through testing again and it's looking good!  Feedback still very much welcome.

File: content/en/apps/guides/forum-offline.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ## Overview
 
-In this post we look at setting up an instance of the [CHT server](https://communityhealthtoolkit.org/) that is initially online so it can be configured, but ultimately is deployed fully offline.  We'll be using the information from the newly published [Offline CHT Server](https://docs.communityhealthtoolkit.org/apps/guides/offline/) guide in the CHT docs as well as the existing guide on [how to deploy a self-hosted CHT server](https://github.com/medic/cht-infrastructure/tree/master/self-hosting).
+In this post we look at setting up an instance of the [CHT server](https://communityhealthtoolkit.org/) that is initially online so it can be configured, but ultimately is deployed fully offline.  We'll be using the information from the newly published [Offline CHT Server](hhttps://docs.communityhealthtoolkit.org/apps/guides/hosting/offline/) guide in the CHT docs as well as the existing guide on [how to deploy a self-hosted CHT server](https://docs.communityhealthtoolkit.org/apps/guides/hosting/self-hosting/).
```
Comment (michaelkohn) [inline content/en/apps/guides/forum-offline.md:3]: `[Offline CHT Server](hhttps://docs.communityhealthtoolkit.org/apps/guides/hosting/offline/)`

There's an extra `h` in  `hhttps` :-)

**Comment by mrjones-plip** [comment]
Forum [post is live](https://forum.communityhealthtoolkit.org/t/prototyping-a-fully-offline-cht-server-environment/1232), closing PR

**Pull request closed without merging**